ui/ui-app/src/utils/rest.utils.ts

```diff
@@ -102,6 +104,24 @@ export function createOptions(headers: { [header: string]: string }): AxiosReque
     return { headers };
 }
 
+/**
+ * Creates the request Auth options used by the HTTP service when making API calls.
+ * @param auth
+ */
+export async function createAuthOptions(auth: AuthService): Promise<AxiosRequestConfig> {
```

Comment:
Does this need to be async?  Seems like it doesn't actually return a promise.... except in the `else` path.

Reply:
It needs to be async in case of Oidc ... (where there is an additional http call), do you prefer to turn this in another way?

Reply:
Ah yes.  I looked for this but missed the call to `auth.getToken()`.  All good.